include/visit_struct/visit_struct.hpp

```diff
@@ -141,8 +252,7 @@ struct visitable<STRUCT_NAME, void> {
     using self_type = STRUCT_NAME;                                                                 \
     VISIT_STRUCT_PP_MAP(VISIT_STRUCT_MEMBER_HELPER_TYPE, __VA_ARGS__)                              \
   }                                                                                                \
-                                                                                                   \
-  static constexpr bool value = true;                                                              \
+  static VISIT_STRUCT_CONSTEXPR const bool value = true;                                           \
```

Comment:
- I used visual 2013 with doesn't have `consexpr`, so I had to change it.
- I didn't test intrusive header.